apps/i3wm/i3wm.py

```diff
@@ -29,6 +63,44 @@ def i3wm_shell():
         key = settings.get("user.i3_mod_key")
         actions.key(f"{key}-enter")
 
+    def i3wm_focus(what: str):
+        """Move focus"""
+        subprocess.check_call(("i3-msg", "focus", what))
+
+    def i3wm_switch_to_workspace(which: Union[str, int]):
+        """Focus the specified workspace"""
+        if isinstance(which, int):
+            subprocess.check_call(("i3-msg", "workspace", "number", str(which)))
```

Comment:
Should we just override `user.desktop(number: int)`?

Noteworthy is that the desktop stuff is in `plugin/desktops/`. If this starts overriding those actions, should it live in `core/`?

Reply:
I suppose the other alternative to overriding is to check whether we're in i3 in the Linux implementation, but that seems a bit awkward to muddy the common implementation with.

Reply:
I reckon leave off this kind of thing for another PR to keep this one focused on just removing the system_command usage with no usage changes.